dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -69,6 +69,33 @@ def SetUp(
         reason, grr_server_url, grr_username, grr_password,
         approvers=approvers, verify=verify)
 
+  def _SeenLastMonth(self, timestamp):
+    """Take a UTC timestamp and check if it is in the last month.
+
+    Args:
+      timestamp (int): A timestamp in microseconds.
+
+    Returns:
+      boolean: True if the timestamp is in last month from now.
+    """
+    last_seen_datetime = datetime.datetime.utcfromtimestamp(
+        timestamp / 1000000)
+    # 30 days before now()
+    month_ago = datetime.datetime.utcnow() - datetime.timedelta(30)
+    return  last_seen_datetime > month_ago
+
+  def _ActiveLastMonth(self, result):
```

Comment:
I'd rename this to `_FilterActiveClients`

Reply:
done